content/en/ddsql_editor/reference/expressions_and_operators.md

```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
 
 For tag references and tag groups, the equality operator (`=`) is treated as a "contains" comparison. See the [Querying Tags in DDSQL][1] for more details.
 
-Additionally, DDSQL supports the following SQL keywords, which function as standard boolean operators:
+## SQL comparison keywords
 
-- `NOT`
-- `AND`
-- `OR`
+DDSQL supports the following SQL keywords, which function as standard Boolean operators:
+
+| Operator | Description            | Example | Result |
+|----------|------------------------|---------|--------|
+| `NOT`    | Filter records based on more than one condition. | `SELECT * FROM Student WHERE NOT Major = 'Economics';`   | Return all students that do not major in Economics.  |
+| `AND`    | Filter records based on more than one condition. | `SELECT * FROM Student WHERE Age = 18 AND Major = 'Economics';`   | Return all students that are age 18 AND major in Economics.  |
+| `OR`     | Filter records based on more than one condition. | `SELECT * FROM Student WHERE Age = 18 OR Major = 'Economics';`   | Return all students that are either age 18 OR major in Economics.  |
 
 DDSQL also supports the following comparator keywords as they are defined in the SQL standard:
 
-- `IS NULL`
-- `IS NOT NULL`
-- `LIKE`
-- `NOT LIKE`
-- `IN`
-- `NOT IN`
+| Operator     | Description            | Example | Result |
+|--------------|------------------------|---------|--------|
+| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM albums WHERE artist IS NULL`   | Return all rows that contain no data in the `artist` column.  |
```

Comment:
select *
from host
where cloud_provider IS NULL

Reply:
```suggestion
| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM host WHERE cloud_provider IS NULL`   | Return all rows that contain no data in the `cloud_provider` column.  |
```